_posts/2019-03-29-weather-app.html

```diff
@@ -0,0 +1,25 @@
+---
+title: Weather App
+thumbnail: -SITE_URL-/images/dark-sky-thumbnail.png
+excerpt: "A Weather app that uses an express backend to relay weather data from the DarkSky API to a React frontend."
+githublink: "https://github.com/SDries/sdries-darksky"
+# bitbucketlink: "#"
+demolink: "https://sdries-darksky.herokuapp.com"
+---
+<p>
+	This application allows a user to view the observed (in the past 30 days) or forecasted (in the future) daily weather conditions for a given location (the user's current location).
+</p>
+<p>
+	In the interest of continuity I will document all of the technical aspects of the application on the github page.
+	Rather, be sure to read the readme files in the source code. There's one file pertaining to the back- and one to the frontend of the application.
```

Comment:
Deze zin is nogal wonky, wat wil je ermee zeggen? Voor zover ik 't snap, lijkt 't me beter om deze te gebruiken. "I will document all of the technical aspects of the application on the Github page. However, the most important files to understand what I did here, are the readme files in the source code."